qiita_db/metadata_template.py

```diff
@@ -760,6 +760,13 @@ def _check_special_columns(cls, md_template, obj):
             The obj to which the metadata template belongs to. Study in case
             of SampleTemplate and RawData in case of PrepTemplate
         """
+        # Check disallowed col names
+        disallowed = {'study_id', 'processed_data_id'}
```

Comment:
Actually, this is already violated on the metadata_template.py file, so we need to change that ASAP unless we are droppng that directive and creating circular import fun again.